scripts/common.py

```diff
@@ -42,8 +42,8 @@
 ISORT_VERSION = '5.8.0'
 PYCODESTYLE_VERSION = '2.6.0'
 PSUTIL_VERSION = '5.8.0'
-PYLINT_VERSION = '2.8.3'
-PYLINT_QUOTES_VERSION = '0.1.9'
+PYLINT_VERSION = '2.11.1'
+PYLINT_QUOTES_VERSION = '0.2.3'
```

Comment:
Any specific reason for this change?

Reply:
I was trying to get this working. I'll revert them.